packages/vaadin-time-picker/src/vaadin-time-picker.js

```diff
@@ -85,15 +85,13 @@ class TimePickerElement extends ElementMixin(ControlStateMixin(ThemableMixin(Pol
       </style>
       <vaadin-combo-box-light
         allow-custom-value=""
-        item-label-path="value"
```

Comment:
Removing this property doesn't change anything so far as by default `item-label-path === "label"` ([proof](https://github.com/vaadin/web-components/blob/1eb70d0d9ffecbb95f0fed4a875e60ceec813a33/packages/vaadin-combo-box/src/vaadin-combo-box-mixin.js#L176)) and `item.value === item.label` ([proof](https://github.com/vaadin/web-components/blob/70e820ca6ad22d0a85a4b50bfad65276ed36f344/packages/vaadin-time-picker/src/vaadin-time-picker.js#L559)).